includes/collection/class-interactions.php

```diff
@@ -229,7 +229,7 @@ public static function allowed_comment_html( $allowed_tags, $context = '' ) {
 	 */
 	public static function activity_to_comment( $activity ) {
 		$comment_content = null;
-		$actor           = object_to_uri( $activity['actor'] );
```

Comment:
I think it should fail if there is no actor!

Reply:
It fails a few lines later, in `get_remote_metadata_by_actor()`